Modules/Core/SpatialObjects/include/itkSpatialObjectPoint.h

```diff
@@ -73,56 +81,86 @@ class ITK_TEMPLATE_EXPORT SpatialObjectPoint
       {
       firstCoordinate, static_cast<double>(otherCoordinate)...
       };
-    m_X = coordinates;
+    m_PositionInObjectSpace = coordinates;
     }
 
+  /** Return a pointer to the point object. */
+  const PointType & GetPositionInObjectSpace() const
+  { return m_PositionInObjectSpace; }
+
+  void SetSpatialObject( SpatialObjectType * so )
```

Comment:
Call `Modified()`?

Reply:
As above, I think Points should remain non-ITK classes.  They are simply nearly atomic data containers.